pintail/site.py

```diff
@@ -1662,7 +1662,7 @@ def __init__(self, site, filename):
                                  stdout=subprocess.PIPE,
                                  env=site.get_script_env(self),
                                  universal_newlines=True)
-            self._configscr.readfp(p.stdout)
+            self._configscr.read(p.stdout)
```

Comment:
According to https://gitlab.gnome.org/Teams/Websites/help.gnome.org/-/blob/master/.gitlab-ci/fedora.Dockerfile#L35, this should work:

```suggestion
            self._configscr.read_file(p.stdout)
```